releases/stellar/2025-02-GMP-v1.1.0.md

```diff
@@ -0,0 +1,212 @@
+# Stellar GMP v1.1.0
+
+|                | **Owner**                            |
+| -------------- | ------------------------------------ |
+| **Created By** | @nbayindirli (<noah@interoplabs.io>)       |
+| **Deployment** | @RiceAndMeet (<steven@interoplabs.io>) |
+
+| **Network**          | **Deployment Status** | **Date** |
+| -------------------- | --------------------- | -------- |
+| **Devnet Amplifier** | -                     | TBD      |
+| **Stagenet**         | -                     | TBD      |
+| **Testnet**          | -                     | TBD      |
+| **Mainnet**          | -                     | TBD      |
+
+- [v1.1.0 Crates](https://crates.io/crates/stellar-axelar-gateway/1.1.0)
+- [v1.1.0 Releases](https://github.com/axelarnetwork/axelar-cgp-stellar/releases/tag/stellar-axelar-gateway-v1.1.0)
+
+## Background
+
+- This is the v1.1.0 Stellar GMP release.
+
+Contract changes in the release:
+
+1. Add contractstorage attribute macro, enabling named keys, typed values, storage method specification in the storage layout, storage schema validation, and built-in TTL extension. Use this macro in lieu of Soroban's contracttype macro on storage enums and calling storage on the env.
```

Comment:
Mention that there are no breaking changes, and highlight that TTL will be extended automatically whenever storage is accessed, which is a public facing change